apps/etterlatte-saksbehandling-ui/client/src/shared/tilbakemelding/Tilbakemelding.tsx

```diff
@@ -5,33 +5,109 @@ import { NoeytralEmoji } from '~shared/tilbakemelding/emoji/NoeytralEmoji'
 import { GladEmoji } from '~shared/tilbakemelding/emoji/GladEmoji'
 import { EkstatiskEmoji } from '~shared/tilbakemelding/emoji/EkstatiskEmoji'
 import { ClickEvent, trackClickMedSvar } from '~utils/amplitude'
-import { useState } from 'react'
+import { JSX, useState } from 'react'
 import {
   leggTilbakemeldingGittILocalStorage,
   tilbakemeldingForBehandlingEksisterer,
 } from '~shared/tilbakemelding/tilbakemeldingLocalStorage'
 
-enum TilbakemeldingSvar {
-  VELDIG_MISFORNOEYD = 'veldig misfornøyd',
-  MISFORNOEYD = 'misfornøyd',
-  NOEYTRAL = 'nøytral',
-  FORNOYED = 'fornøyd',
-  VELDIG_FORNOYED = 'veldig fornøyd',
+enum AlternativEmoji {
+  SINNA = 'SINNA',
+  LEI = 'LEI',
+  NOEYTRAL = 'NOEYTRAL',
+  GLAD = 'GLAD',
+  EKSTATISK = 'EKSTATISK',
 }
 
+type Alternativ = {
+  emoji: AlternativEmoji
+  tekstVisning: string
+  tekstEvent: string
+}
+
+const emojiForAlternativ: Record<AlternativEmoji, JSX.Element> = {
+  [AlternativEmoji.EKSTATISK]: <EkstatiskEmoji />,
+  [AlternativEmoji.GLAD]: <GladEmoji />,
+  [AlternativEmoji.LEI]: <LeiEmoji />,
+  [AlternativEmoji.NOEYTRAL]: <NoeytralEmoji />,
+  [AlternativEmoji.SINNA]: <SinnaEmoji />,
+}
+
+const FORNOEYD_MISFORNOEYD_ALTERNATIVER: Alternativ[] = [
+  {
+    emoji: AlternativEmoji.SINNA,
+    tekstVisning: 'Veldig misfornøyd',
+    tekstEvent: 'veldig misfornøyd',
+  },
+  {
+    emoji: AlternativEmoji.LEI,
+    tekstVisning: 'Misfornøyd',
+    tekstEvent: 'misfornøyd',
+  },
+  {
+    emoji: AlternativEmoji.NOEYTRAL,
+    tekstVisning: 'Nøytral',
+    tekstEvent: 'nøytral',
+  },
+  {
+    emoji: AlternativEmoji.GLAD,
+    tekstVisning: 'Fornøyd',
+    tekstEvent: 'fornøyd',
+  },
+  {
+    emoji: AlternativEmoji.EKSTATISK,
+    tekstVisning: 'Veldig fornøyd',
+    tekstEvent: 'veldig fornøyd',
+  },
+] as const
+
+const ENIG_UNIG_ALTERNATIVER: Alternativ[] = [
+  {
+    emoji: AlternativEmoji.SINNA,
+    tekstVisning: 'Helt uenig',
+    tekstEvent: 'helt uenig',
+  },
+  {
+    emoji: AlternativEmoji.LEI,
+    tekstVisning: 'Uenig',
+    tekstEvent: 'uenig',
+  },
+  {
+    emoji: AlternativEmoji.GLAD,
+    tekstVisning: 'Enig',
+    tekstEvent: 'enig',
+  },
+  {
+    emoji: AlternativEmoji.EKSTATISK,
+    tekstVisning: 'Helt enig',
+    tekstEvent: 'helt enig',
+  },
+] as const
```

Comment:
Trenger vi egt å styre med masse records, typer og mapping? 😅 Kan man ikke bare lage 2 komponenter med litt duplikat kode and call it a day.

Reply:
Aight jeg kan bare scrappe de gamle typene egentlig